tests/components/rfxtrx/test_light.py

```diff
@@ -70,62 +74,67 @@ async def test_one_light(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    import RFXtrx as rfxtrxmod
+    state = hass.states.get("light.test")
+    assert state
+    assert state.state == "off"
+    assert state.attributes.get("friendly_name") == "Test"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)